src/components/Modal.js

```diff
@@ -34,7 +44,7 @@ const ModalPanel = styled.div`
   box-shadow: 0px 0px 60px 1px rgba(0, 0, 0, 0.4);
   pointer-events: auto;
 
-  @media (max-width: 730px) {
+  @media (max-width: ${px(MODAL_WIDTH + 30)}) {
```

Comment:
@LucianBuzzo How about exposing a `width` property and fallback to `MODAL_WIDTH` when `undefined`?
For example [this wireframe](https://app.zeplin.io/project/59ccc17f31f59fb5e145a3ac/screen/5a12fe6efe34e34e9dd215b5) uses 1000px wide modals.

Reply:
@thgreasi Yeah that sounds great

Reply:
`calc` doesn't work inside media queries, changed that to use `max-width` instead.